domain/src/main/java/org/fao/geonet/domain/MetadataDraft.java

```diff
@@ -0,0 +1,38 @@
+package org.fao.geonet.domain;
+
+import java.io.Serializable;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Table;
+
+import org.apache.lucene.document.Document;
+import org.fao.geonet.entitylistener.MetadataDraftEntityListenerManager;
+
+/**
+ * This is a normal {@link Metadata} but on its draft version.
+ * 
+ * Privileges, categories and non-XML properties will not be stored.
+ *
+ * @author María Arias de Reyna
+ */
+@Entity
+@Table(name = MetadataDraft.TABLENAME)
+@Access(AccessType.PROPERTY)
+@EntityListeners(MetadataDraftEntityListenerManager.class)
+public class MetadataDraft extends AbstractMetadata implements Serializable {
```

Comment:
I use UUID as reference to link between draft and metadata, because UUID is what is used on the API. If you think you can use that FK for something in the future, sure.

Reply:
It is more to have a constraint on the db that does the check to avoid dummy data in db.

Reply:
ok, adding FK.